target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4029-mr74.dts

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Device Tree Source for Meraki MR74 (Ladybug)
+
+#include "qcom-ipq4029-insect-common.dtsi"
+
+/ {
+	model = "Meraki MR74 Access Point";
+	compatible = "meraki,mr74";
+};
```

Comment:
Since I was involved in the MR33 port. I looked if I had the old files still around. This is a diff between the MR33 and MR74 device-tree from
"25-201610270754-Gd75eda32-Lccc6777d-aacharya".

It looks like the RGB-LED controller needs a different enable-gpio?
Do you know about this?

--- fdt@1.dts	2018-02-24 17:19:46.912101308 +0100
+++ fdt@3.dts	2018-02-24 17:19:53.155450521 +0100
@@ -3,7 +3,7 @@
 / {
 	#address-cells = <0x1>;
 	#size-cells = <0x1>;
-	model = "Meraki MR33";
+	model = "Meraki MR74";
 	compatible = "qcom,ipq40xx-apdk04.1", "qcom,ipq40xx";
 	interrupt-parent = <0x1>;
 
@@ -1677,7 +1677,7 @@
 				compatible = "ti,lp5562";
 				clock-mode = [02];
 				reg = <0x30>;
-				enable-gpio = <0x51 0x30 0x0>;
+				enable-gpio = <0x51 0xe 0x1>;
 
 				chan0 {
 					chan-name = "tricolor0:red";

Reply:
Thanks, well spotted. I have added a tricolor label with `enable-gpio` specified in each of the MR33/MR74 dts files. Let me know if you prefer I do that another way.